addons/web/controllers/binary.py

```diff
@@ -100,7 +100,7 @@ def content_assets(self, id=None, filename=None, unique=False, extra=None, nocac
 
         send_file_kwargs = {'as_attachment': False}
         if unique:
-            send_file_kwargs['max_age'] = http.STATIC_CACHE_LONG
```

Comment:
should we remove it or keep both?
https://caniuse.com/?search=immutable
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control#immutable

Reply:
In http.py if there is immutable then the max-age is set automatically.